pandas/tests/resample/test_resample_api.py

```diff
@@ -247,7 +247,7 @@ def test_agg_consistency():
                                     check_stacklevel=False):
         expected = r[['A', 'B', 'C']].agg({'r1': 'mean', 'r2': 'sum'})
         result = r.agg({'r1': 'mean', 'r2': 'sum'})
-    assert_frame_equal(result, expected)
+    assert_frame_equal(result, expected, check_like=True)
```

Comment:
LGTM. This could maybe be `check_like=not compat.PY36` so that we continue to check it for 3.6 and above. But I suspect we have other tests ensuring the order.

Reply:
yeah just saw a bunch of failures here.

Reply:
Yeah, I think ideally we should only do this for py 3.5